.travis.yml

```diff
@@ -6,13 +6,105 @@ python:
   - "3.6"
   - "3.7"
   - "3.8"
-  - "pypy3"
+
```

Comment:
Why did you drop PyPy?

Reply:
AFAIK I don't think we call it out in the readme/metadata that we do support it, but you're right, it should be left in. Fixed.